code/iaas/allocator/src/main/java/io/cattle/platform/allocator/service/AllocatorService.java

```diff
@@ -19,5 +20,9 @@
 
     void ensureResourcesReservedForStart(Instance instance);
 
+    void newPortAllocateForLoadBalancerUpdate(Instance instance, List<Port> newPorts);
```

Comment:
Lets change the methods names to:

allocatePortsForInstanceUpdate
releasePortsForInstanceUpdate